test/runners/javascript_spec.js

```diff
@@ -585,6 +585,33 @@ describe( 'javascript runner', function(){
                         done();
                     });
             });
+
+            describe('Fix #259', function() {
+                it('should prevent global reassignment', function(done) {
```

Comment:
Should have named the test case: "should prevent global reassignment cheat"

Before:
```
<DESCRIBE::>Fake test suite
<IT::>fake test
<PASSED::>Test Passed
<COMPLETEDIN::>
<COMPLETEDIN::>4
```

After:
```
<ERROR::>Error: global.global was reassigned<:LF:>           at Object.handleError<:LF:>        at ContextifyScript.Script.runInThisContext<:LF:>    at Object.exports.runInThisContext<:LF:>
```